pyproject.toml

```diff
@@ -313,3 +314,32 @@ ignore_directives = [
     "tab-set",
 ]
 ignore_messages = "^.*(Unknown target name|Undefined substitution referenced)[^`]*$"
+
+[tool.mypy]
+ignore_errors = true
+scripts_are_modules = true
+
+[[tool.mypy.overrides]]
+module = ['mne.evoked', 'mne.io']
+ignore_errors = false
+# Ignore "attr-defined" until we fix stuff like:
```

Comment:
@hoechenberger feel free to look here at what I had to add, basically it gives us a todo list to tackle at some point when people are motivated (or things to live with)